components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -213,6 +216,13 @@ func (c *certRotateFlow) certRotate(cmd *cobra.Command, args []string, flagsObj
 			return errors.New("Error occured while fetching current certs.")
 		}
 
+		if flagsObj.timeout < DEFAULT_TIMEOUT {
+			return errors.Errorf("The operation timeout duration for each individual node during the certificate rotation process should be set to a value greater than %v seconds.", DEFAULT_TIMEOUT)
+		}
+
+		sshConfig.timeout = flagsObj.timeout
+		sshUtil.setSSHConfig(sshConfig)
```

Comment:
should we warn the user for less than the default timeout instead of blocking?

Reply:
I think we should block the user.